packages/core/src/tracing/sentrySpan.ts

```diff
@@ -188,6 +195,8 @@ export class SentrySpan implements Span {
       profile_id: this._attributes[SEMANTIC_ATTRIBUTE_PROFILE_ID] as string | undefined,
       exclusive_time: this._attributes[SEMANTIC_ATTRIBUTE_EXCLUSIVE_TIME] as number | undefined,
       measurements: timedEventsToMeasurements(this._events),
+      is_segment: this._isStandaloneSpan || undefined,
+      segment_id: this._isStandaloneSpan ? this._spanId : undefined,
```

Comment:
m: I think this should be:

```suggestion
      is_segment: (this._isStandaloneSpan && getRootSpan(this) === this) || undefined,
      segment_id: (this._isStandaloneSpan && getRootSpan(this) === this) ? this._spanId : undefined,
```

Reply:
actually wait no, it should be:

```suggestion
      is_segment: (this._isStandaloneSpan && getRootSpan(this) === this) || undefined,
      segment_id: this._isStandaloneSpan ? getRootSpan(this).spanContext().spanId : undefined,
```

as if we are sending (theoretically) a standalone span that is not the root span, then it should have the root span id as segment_id!